spec/features/bulk_upload_lettings_logs_spec.rb

```diff
@@ -22,11 +27,7 @@
   # rubocop:disable RSpec/AnyInstance
```

Comment:
do we need this rubocop disable? I'm assumign it's for the `allow(FormHandler.instance).to receive` but when I check for other instances of that around the codebase they don't need the disable.

Reply:
I think it's for the `allow_any_instance_of(Forms::BulkUploadLettings::UploadYourFile).to receive(:`).and_return("text/csv")`